SYCL/lit.cfg.py

```diff
@@ -269,7 +269,7 @@
 if 'acc' in config.target_devices.split(','):
     found_at_least_one_device = True
     lit_config.note("Tests accelerator device")
-    acc_run_substitute = " env SYCL_DEVICE_FILTER={SYCL_PLUGIN}:acc ".format(SYCL_PLUGIN=config.sycl_be)
+    acc_run_substitute = " env SYCL_DEVICE_FILTER=acc "
```

Comment:
It seems HOST should not be added for the ACC_RUN_PLACEHOLDER because currently HOST is higher heuristic point than ACC.
So, if we add host, the device_selectors will choose HOST instead of ACC.